### PR TITLE
fix bug to make select 1 work

### PR DESCRIPTION
**Pull request by PangZhi**
### Description and related tickets, documents
Fix the bug in TableNameReplacer to make "select 1" work
related jira [PE-1698]

Reviewers: @Huandao0812 @nhanitvn @hai-adatao 

### How to test
UT, client test

### PR Progress
Make sure all checkboxes below are checked before merged
- [x] Branch is in format `prefix/description` (see [this](http://www.guyroutledge.co.uk/blog/git-branch-naming-conventions/))
- [x] Merge check has no conflicts. PR checks passed.
- [x] Code review is done.

### CODE REVIEW

File: spark/src/test/java/io/ddf/spark/TableNameReplacerTests.java [hunk 1]
```diff
@@ -447,29 +429,35 @@ public void BatchTestRelationalOps() {
...
+      String sqlcmd = "select 1";
+      try {
+        tableNameReplacer.run(parser.parse(new StringReader(sqlcmd)));
+      } catch (Exception e) {
+        assert false;
+      }
+    }
 
     @BeforeClass
     public static void startServer() throws Exception {
```
Comment (nhanitvn) [inline spark/src/test/java/io/ddf/spark/TableNameReplacerTests.java]: Should this function name be a more relevant one, such as createDDFManager or initDDFManager? :)

**Comment by hai-adatao** [comment]
Where is the "select 1" unit test? Does this need change in DDF-JDBC also?

File: spark/src/test/java/io/ddf/spark/TableNameReplacerTests.java [hunk 2]
```diff
@@ -447,29 +429,35 @@ public void BatchTestRelationalOps() {
...
             statement = parser.parse(new StringReader(sqlcmd));
         } catch (JSQLParserException e) {
-            e.printStackTrace();
             assert(false);
         }
     }
 
+    @Test
+    public void testSelectOne() {
+      String sqlcmd = "select 1";
```
Comment (nhanitvn) [inline spark/src/test/java/io/ddf/spark/TableNameReplacerTests.java]: @hai-adatao This is the SELECT 1 testcase :D

Comment (hai-adatao) [inline spark/src/test/java/io/ddf/spark/TableNameReplacerTests.java]: :D My bad

File: spark/src/test/java/io/ddf/spark/TableNameReplacerTests.java (hunk 1 above)
Comment (hai-adatao) [inline spark/src/test/java/io/ddf/spark/TableNameReplacerTests.java]: Agree, but this is white line, not in this PR scope :D let's do it while refactoring the tests

**Pull request merged**